utils/rpms/packaging/Dockerfile.centos.7

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018-2022, Intel Corporation
+# Copyright 2018-2021, Intel Corporation
```

Comment:
This is reverting an previously unnecessary copyright update for a file that was not actually modified in 2022.  Per discussion, we should only be modifying the copyright on files when we actually modify them.